lib/messaging/messagehandler.rb

```diff
@@ -32,12 +40,15 @@ def initialize
     @message_handlers << MessageHandlerParameter.new
     @message_handlers << MessageHandlerSchedule.new
     @message_handlers << MessageHandlerStatus.new
+    @message_handlers << MessageHandlerStatus.new
+    @message_handlers << MessageHandlerMessage.new
```

Comment:
I noticed we didn't have a handler for message type 'message'. Added it in here.